### ISSUE TITLE
HoloPicker Size parameters not in documentation

### ISSUE THREAD

**Issue by devarishi7** [LarsWerkman/HoloColorPicker#91]
Creating an App using 2 Holopickers, i accidentally stumbled upon some resizers for the picker (in another issue about the SVBar) Being : 

                app:color_center_halo_radius="40dp"
                app:color_center_radius="30dp"
                app:color_pointer_halo_radius="22dp"
                app:color_pointer_radius="16dp"
                app:color_wheel_radius="70dp"
                app:color_wheel_thickness="24dp"

for instance then i started looking at for info on how to do something similar for the 'bars' but google didn't find an answer. but somehow i managed to find : 

                app:bar_length="140dp"
                app:bar_pointer_halo_radius="14dp"
                app:bar_pointer_radius="10dp"
                app:bar_thickness="8dp"

i mus have been looking through the source-code, but that is a tad tricky for me, Java is not my first language and AS is taking up so much memory that many times webpages get put in the pagefile on the harddrive, which makes it all rather slow.
Anyway the question i did not find any answer to is how can i remove the margins from the picker (or make them smaller) so that i can put them a little closer together while using "Constraint-layout".
Basically anyway are there any other parameters i can modify.
